dspace-api/src/main/java/org/dspace/eperson/SubscribeCLITool.java

```diff
@@ -316,7 +316,7 @@ public static void main(String[] argv) {
         boolean test = line.hasOption("t");
 
         if (test) {
-            log.setLevel(Level.DEBUG);
+            //TODO Replace with log4j2: org.apache.logging.log4j.core.config.Configurator.setLevel();
```

Comment:
Looks like this relates to previous comment, and we seem to not have updated the old code here?

Reply:
I think we should abjure such black magic and just drop this whole `if`.